functests/ptp/ptp.go

```diff
@@ -160,6 +175,52 @@ var _ = Describe("ptp", func() {
 	})
 })
 
+func checkPolicies(namespace string) {
+	var masters []ptpv1.PtpConfig
+	var slaves []ptpv1.PtpConfig
+
+	configList, err := client.Client.PtpConfigs(namespace).List(metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	for _, config := range configList.Items {
+		for _, profile := range config.Spec.Profile {
+			if *profile.Ptp4lOpts == "" && *profile.Phc2sysOpts == "-a -r -r" {
```

Comment:
Can you make this independent on the order?

Reply:
Done